packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1656,17 +1744,34 @@ export class Combobox
   }
 
   private renderListBoxOptions(): JsxNode {
-    return this.filteredItems.map((item) => (
-      <li
-        ariaLabel={item.label}
-        ariaSelected={item.selected}
-        id={item.guid ? `${itemUidPrefix}${item.guid}` : null}
-        role="option"
-        tabIndex="-1"
-      >
-        {item.heading || item.textLabel}
-      </li>
-    ));
+    const selectAllComboboxItem = this.selectAllEnabled &&
+      this.selectionMode !== "single" &&
+      this.selectionMode !== "single-persist" && (
+        <li
+          ariaLabel={this.messages.selectAll}
+          ariaSelected={this.allSelected}
+          id={`${this.guid}-select-all-enabled-screen-reader`}
+          tabIndex="-1"
+          value="select-all"
```

Comment:
fixme: I don't think `value` is a valid property of a list item.

Reply:
fixme: I think this needs a ` role="option"` just like the other list item below.

Reply:
The other list item below also has text inside the list item. Should this have text inside? Maybe the ariaLabel isn't necessary if it has text inside?